packages/phone-number-privacy/signer/src/signing/query-quota.ts

```diff
@@ -90,8 +90,13 @@ async function getQueryQuota(account: string, hashedPhoneNumber?: string) {
     )
   }
 
-  const accountBalance = await getDollarBalance(account)
-  if (accountBalance.isGreaterThanOrEqualTo(config.quota.minDollarBalance)) {
+  const cUSDAccountBalance = await getDollarBalance(account)
+  const celoAccountBalance = await getCeloBalance(account)
+  // Min balance can be in either cUSD or CELO
+  if (
+    cUSDAccountBalance.isGreaterThanOrEqualTo(config.quota.minDollarBalance) ||
+    celoAccountBalance.isGreaterThanOrEqualTo(config.quota.minCeloBalance)
+  ) {
```

Comment:
Nit: i would be more efficient to check them individually, that way we short circuit the 2nd state read in the case of wallets with min cUSD balance.

Reply:
Good point. What do you think about running both checks in parallel? This would optimize for the 0 cUSD balance case as well.

Reply:
Yeah `Promise.all` would work similarly well but we still double the forno load per request. I'm sure it's fine either way :)

Reply:
Thanks! Went with the promise.all approach.